iOS Client/State/MBState.swift

```diff
@@ -67,11 +67,13 @@ struct MBDevotionState: DevotionState {
 protocol PodcastsState {
     var podcasts: Loaded<[MBPodcast]> { get set }
     var selectedPodcast: MBPodcast? { get set }
+    var player: PlayerState { get set }
 }
 
 struct MBPodcastsState: PodcastsState {
     var podcasts: Loaded<[MBPodcast]> = .initial
     var selectedPodcast: MBPodcast? = nil
+    var player: PlayerState = .initialized
 }
 
```

Comment:
hmm, maybe it wasn't the best idea on my part to protocol out all this stuff . . . but that's neither here nor there for this PR. I'd be curious to know if yall use protocols at work for your state

Reply:
yeah protocols here seem a little bit unnecessary, I don't think they're really serving us any purpose. We don't use them on our state